src/Task/Scheduler.php

```diff
@@ -68,6 +68,8 @@ public function __construct(
     }
 
     /**
+     * TODO key see createTaskAndSchedula
```

Comment:
the key can be added by the task-builder therefor we can remove this todo. @alexander-schranz